applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -122,6 +122,22 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 	int ret;
 	struct ble_iso_data *iso_received = NULL;
 
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+	static uint32_t packet_count;
+
+	if (channel_index == AUDIO_CH_R) {
+		packet_count++;
+		if ((packet_count % 1000) == 0) {
+			LOG_INF("Packets received from right channel: %d", packet_count);
+		}
+
+		return;
+	} else if (channel_index != AUDIO_CH_L) {
```

Comment:
I think this is a tad hard to read. Left channel checks should be first, then right, then an "else". We can also use a switch case.

Reply:
Switched to switch case